components/table/ProjectTable.tsx

```diff
@@ -21,12 +21,12 @@ function ProjectTable({ projects }: ProjectListProps): JSX.Element {
                         </Tr>
                     </Thead>
                     <Tbody>
-                        {projects.map((el, idx) => {
```

Comment:
`el` is a non-idiomatic varibale name, something like `project` is much easier to read without context and should be preferred